### ISSUE TITLE
Input does not compile with latest version of std/async

### ISSUE THREAD

**Issue by joyzheng** [keegandonley/input-deno#24]
**Describe the bug**
std/async removed `deferred` in https://github.com/denoland/deno_std/pull/3852

Since input does not pin the version of `std/async`, this results in the following error when trying to import the input library: 

```
Warning Implicitly using latest version (0.210.0) for https://deno.land/std/async/mod.tsiB (217/258)
error: Uncaught SyntaxError: The requested module 'https://deno.land/std/async/mod.ts' does not provide an export named 'deferred'
import { deferred } from "https://deno.land/std/async/mod.ts";
         ^
    at <anonymous> (https://deno.land/x/input@2.0.3/index.ts:5:10)
```

**To Reproduce**

Attempt to import the `input` package in deno

**Expected behavior**

The import succeeds

**Program Output**
n/a

**Versions (please complete the following information):**
 - OS: Mac OSX 14.2.1
 - Deno Version : 1.32.4

**Additional context**

**Comment by keegandonley** [comment]
Thank you for your report. I'll plan to release version 2.1 shortly to remove the use of `deferred`. I won't be able to until the evening here, so if you'd like to raise a PR in the meantime to pin the version it would be much appreciated.

**Comment by joyzheng** [comment]
Thanks, very much appreciate it! (unfortunately I won't be able to get around to making a PR until later today either)

**Comment by keegandonley** [comment]
@joyzheng I've released version 2.0.4 that pins std at `0.205.0` to allow this to compile and run. I still plan to remove the use of the deprecated `deferred` once I have some more time.
